helm/observability-operator/values.yaml

```diff
@@ -15,6 +15,12 @@ managementCluster:
   pipeline: pipeline
   region: region
 
+alertmanager:
```

Comment:
I would prefer this to be named alerting to match with monitoring and eventually logging

Reply:
Why do we use a slackAppID https://github.com/giantswarm/giantswarm-configs/pull/216 in the secret if it's missing here?

Reply:
Updated this key to alerting.

The slackAppID was mainly for scripting the update of slack token, because a token is bound to an app in slack api. But this value is not needed here.